share/localization/gmt_cn1.locale

```diff
@@ -25,9 +25,9 @@ W	4	
 W	5	������		����	T
 W	6	������		����	F
 W	7	������		����	S
-# The following lines give the translation of the 4 major compass directions (WESN).
-# Name in full (optionally capitalized), abbreviation of longitude and latitude on map edge and wind direction (may be more than one letter).
-C	1	West		W	W
-C	2	East		E	E
-C	3	South		S	S
-C	4	North		N	N
+# ������õ��ͼ�ĸ�����������
+# ����õ��ͼ��ʹ����д�����ֻ�޸����һ�м���
+C	1	West		W	��
+C	2	East		E	��
+C	3	South		S	��
+C	4	North		N	��
```

Comment:
@ZMAlt  I think you need to update both the 3rd and the 5th columns to Chinese.

Reply:
Done